Sw integrate queue with bot

swangs opened the pull request.

swangs committed:
sw - use http for rubygems
Some gems could not be fetched over https. This is currently a stashed change on the instance but I can commit it here

swangs committed:
sw - use text type instead of string
The access and refresh tokens are too long to be stored as strings
Also fixed some typos in the station names and env vars

swangs committed:
sw - add logic for queue messaging
When a spot opens the next person in line will be notified

jmangel commented on app/models/channel_queue.rb:
```diff
@@ -17,6 +17,10 @@ def members_string
   end
 
+  def first_user_in_line
+    channel_queue_memberships.order(created_at: :asc).joins(:user).first&.user
```
There is a NextInQueue service that's also used by the `/queue open` command, but it shouldn't be any different or more robust than this, afaik

jmangel reviewed (approved):
LGTM! Thanks

swangs committed:
sw - use NextInQueue service
The service already exists and contains similar logic so it should be reuseable. I also remembered that the &. operator doesn't work in rails 2 so I have to remove that anyways
